src/test/resources/pageUrl.properties

```diff
@@ -0,0 +1,3 @@
+login.page=https://magento.softwaretestingboard.com/customer/account/create/
+
+
```

Comment:
There is no need for so many properties files. It is better to store in one (at least for your case)
You just need to have appropriate names. Like
magento.login.page or magento.login.url

For sauce lab you can start you properties from sauce.lab

Same will be from browser
browser.explicit.time
browser.remote.url
browser.environment
browser.type


and etc.


Again it all more about supporting code. When you have so many properties you need to remember property file name and property name. When everything in one, you need to remember only property name